src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,10 +25,8 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(
-            ~TargetFrameworkMonikers.Netcoreapp, 
-            "NetFramework: NetEventSource is only part of .NET Core;" + 
-            "UapAot: RemoteExecutorConsoleApp.exe crashes trying to load netstandard.dll - StrongName issue")]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/19909", TargetFrameworkMonikers.Uap)]
```

Comment:
It's still easier to have "click on URL" work consistently instead of having each reader execute the algorithm:

   "If string, click. If number, string.prefix("https://github.com/dotnet/corefx/issues/###")"

Code should be optimized for readers, not authors. 

Though this one's easier on the authoring end too - I create the issue, copy-paste the url from the browser and paste it in without stopping to think "Oh, what repo is this. Oh, it's corefx - have to do extra work so that the reader can do extra work."